src/components/file-upload/index.md

```diff
@@ -108,12 +108,34 @@ For example, ‘You can only select up to 10 files at the same time’.
 
 Say ‘The selected file must use the template’.
 
+## How to turn on accessibility improvements
```

Comment:
Because our changes include other usability improvements, I wonder if we should drop the word 'accessibility' here?

Reply:
Yep I agree with this, I'll make the change.